plugins/utils/tmux-navigator.nix

```
@@ -6,37 +6,127 @@
+
+      disable_when_zoomed = helpers.defaultNullOpts.mkBool false ''
+        By default, if you zoom the tmux pane running vim and then attempt to navigate "past" the edge of the vim session, tmux will unzoom the pane.
+        This is the default tmux behavior, but may be confusing if you've become accustomed to navigation "wrapping" around the sides due to this plugin.
+
+        This option disables the unzooming behavior, keeping all navigation within vim until the tmux pane is explicitly unzoomed.
+      '';
+
+      preserve_zoom = helpers.defaultNullOpts.mkBool false ''
+        As noted in `disable_when_zoomed`, navigating from a vim pane to another tmux pane normally causes the window to be unzoomed.
+        Some users may prefer the behavior of tmux's `-Z` option to `select-pane`, which keeps the window zoomed if it was zoomed.
+
+        This option enables that behavior.
+
+        Naturally, if `disable_when_zoomed` is enabled, this option will have no effect.
+      '';
+
+      no_wrap = helpers.defaultNullOpts.mkBool false ''
+        By default, if you try to move past the edge of the screen, tmux/vim will "wrap" around to the opposite side.
+
+        This option disables "wrapping" in vim, but tmux will need to be configured separately.
+
+        Tmux doesn't have a "no_wrap" option, so whatever key bindings you have need to conditionally wrap based on position on screen:
+
+        ```shell
+          is_vim="ps -o state= -o comm= -t '#{pane_tty}' \
+            | grep -iqE '^[^TXZ ]+ +(\\S+\\/)?g?(view|l?n?vim?x?|fzf)(diff)?$'"
+
+          bind-key -n 'C-h' if-shell "$is_vim" { send-keys C-h } { if-shell -F '#{pane_at_left}'   {} { select-pane -L } }
+          bind-key -n 'C-j' if-shell "$is_vim" { send-keys C-j } { if-shell -F '#{pane_at_bottom}' {} { select-pane -D } }
+          bind-key -n 'C-k' if-shell "$is_vim" { send-keys C-k } { if-shell -F '#{pane_at_top}'    {} { select-pane -U } }
+          bind-key -n 'C-l' if-shell "$is_vim" { send-keys C-l } { if-shell -F '#{pane_at_right}'  {} { select-pane -R } }
+
+          bind-key -T copy-mode-vi 'C-h' if-shell -F '#{pane_at_left}'   {} { select-pane -L }
+          bind-key -T copy-mode-vi 'C-j' if-shell -F '#{pane_at_bottom}' {} { select-pane -D }
+          bind-key -T copy-mode-vi 'C-k' if-shell -F '#{pane_at_top}'    {} { select-pane -U }
+          bind-key -T copy-mode-vi 'C-l' if-shell -F '#{pane_at_right}'  {} { select-pane -R }
+        ```
+      '';
+
+      no_mappings = helpers.defaultNullOpts.mkBool false ''
+        By default `<C-h>`, `<C-j>`, `<C-k>`, `<C-l>`, & `<-\\>` are mapped to navigating left, down, up, right, & previous, respectively.
+
+        This option disables those default mappings being created.
+
+        You can use the plugin's five commands to define your own custom mappings:
+
+        ```nix
+          keymaps = [
+            {
+              key = "<C-w>h";
+              action = "<cmd><C-U>TmuxNavigateLeft<cr>";
```

`<cmd><C-U>TmuxNavigateLeft<cr>` complains "not an editor command".

Looking at [`:h <cmd>`](https://neovim.io/doc/user/map.html#%3CCmd%3E), it seems like `<cmd>` alone does the same thing as `:<C-U>` while also working better with more modes.

The `<C-U>` is intended to clear the command prefix that may exist, e.g. when using `:` while a range is selected in visual mode. It's similar to using `<C-u>` in a terminal.

```suggestion
              action = "<cmd>TmuxNavigateLeft<cr>";
```

---

Also:
> The command is not echo'ed, no need for `<silent>`.